docs-ref-conceptual/msal-based-azure-cli.md

```diff
@@ -14,24 +14,37 @@ keywords: msal, msal-based azure cli
 
 # MSAL-based Azure CLI
 
-Starting in version 2.30.0, Azure CLI uses [MSAL](https://github.com/AzureAD/microsoft-authentication-library-for-python) as the underlying authentication library. MSAL uses AAD v2.0 authentication flow to provide more functionality and increases security for token cache.
+Starting in version 2.30.0, Azure CLI uses [MSAL](https://github.com/AzureAD/microsoft-authentication-library-for-python)
+as the underlying authentication library. MSAL uses AAD v2.0 authentication flow to provide more
+functionality and increases security for token cache.
```

Comment:
Are these line breaks manually added or automatically added by tools? 

If you check other files, we don't break lines as this will make them difficult to be `diff`ed.

Reply:
Done by tools. Habit from PowerShell docs. It doesn't change the website rendering, it just makes it easier to edit on all screens as it breaks lines at 100 characters. Undid the change and will wait for merge :)

Reply:
In VS Code, we can use toggle word wrap easily (<kbd>alt</kbd> + <kbd>z</kbd>) 

![image](https://user-images.githubusercontent.com/4003950/142790571-f84aabe1-018d-46e8-8212-fbf8e4ad8af9.png)